spec/models/user_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user_1) do
+    User.create(
```

Comment:
Do not create record directly use `Factory`.

Reply:
fix